includes/class-wc-beta-tester-version-picker.php

```diff
@@ -0,0 +1,162 @@
+		if ( ! current_user_can( 'install_plugins' ) ) {
+			return;
+		}
+
+		$settings = WC_Beta_Tester::get_settings();
+		$channel = $settings->channel;
+
+		?>
+		<div class="wrap">
+			<div class="wcbt-content-wrap">
+				<h1><?php esc_html_e( 'Available WooCommerce versions', 'woocommerce-beta-tester' ); ?></h1>
+				<form name="wcbt-select-version" class="wcbt-select-version-form" action="<?php echo esc_attr( admin_url( '/index.php' ) ); ?>">
+					<div class="wcbt-versions-wrap">
+						<?php echo $this->get_versions_html( $channel ); // WPCS: XSS ok. ?>
+					</div>
+					<div class="wcbt-submit-wrap">
+						<a href="#wcbt-modal-version-switch-confirm" class="button-primary" id="wcbt-modal-version-switch-confirm"><?php esc_html_e( 'Switch version', 'woocommerce-beta-tester' ); ?></a>
+					</div>
+					<?php wp_nonce_field( 'wcbt_switch_version_nonce' ); ?>
+
+					<script type="text/template" id="tmpl-wcbt-version-switch-confirm">
+						<div class="wc-backbone-modal wc-backbone-modal-beta-tester-version-info">
+							<div class="wc-backbone-modal-content">
+								<section class="wc-backbone-modal-main" role="main">
+									<header class="wc-backbone-modal-header">
+										<h1>
+											<?php
+											esc_html_e( 'Are you sure you want to switch the version of WooCommerce plugin?', 'woocommerce-beta-tester' );
+											?>
+										</h1>
+										<button class="modal-close modal-close-link dashicons dashicons-no-alt">
+											<span class="screen-reader-text"><?php esc_html_e( 'Close modal panel', 'woocommerce-beta-tester' ); ?></span>
+										</button>
+									</header>
+									<article>
+
+										<table class="wcbt-widefat">
+											<tbody>
+											<tr class="alternate">
+												<td class="row-title">
+													<label for="tablecell"><?php esc_html_e( 'Installed Version:', 'woocommerce-beta-tester' ); ?></label>
+												</td>
+												<td><span class="wcbt-installed-version"><?php echo esc_html( $this->current_version ); ?></span></td>
+											</tr>
+											<tr>
+												<td class="row-title">
+													<label for="tablecell"><?php esc_html_e( 'New Version:', 'woocommerce-beta-tester' ); ?></label>
+												</td>
+												<td><span class="wcbt-new-version">{{ data.new_version }}</span></td>
+											</tr>
+											</tbody>
+										</table>
+
+										<div class="wcbt-notice">
+											<p><?php esc_html_e( 'Notice: We strongly recommend you perform the test on a staging site and create a complete backup of your WordPress files and database prior to performing a version switch. We are not responsible for any misuse, deletions, white screens, fatal errors, or any other issue arising from using this plugin.', 'woocommerce-beta-tester' ); ?></p>
+										</div>
+
+										<input type="submit" value="<?php esc_attr_e( 'Switch version', 'woocommerce-beta-tester' ); ?>" class="button-primary wcbt-go" id="wcbt-submit-version-switch" />
```

Comment:
I think it'd look a bit better if this and the next line went into `<footer />`?

Reply:
Yup, but if I add any longer text into footer, it covers the `<article>`, so maybe better to update the styling there?
I moved the buttons to the `<footer>`, though.